internal/contour/cluster_test.go

```diff
@@ -20,6 +20,8 @@ import (
 
 	"github.com/envoyproxy/go-control-plane/envoy/api/v2"
 	"github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
+	envoy_api_v2_core4 "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
+	google_protobuf "github.com/gogo/protobuf/types"
```

Comment:
why is this import renamed, `types` shouldn't conflict with anything in the file.